guides/publish-and-share.qmd

```diff
@@ -117,7 +117,11 @@ Instead of archiving research data in a data repository, you may choose to publi
 ### Data and Software Licensing
```

Comment:
```suggestion
```

Otherwise there will be a "Data Licensing" and "Software Licensing" subheading to "Data and Software Licensing"